manifests/base/jupyter-rocm-pytorch-notebook-imagestream.yaml

```diff
@@ -16,8 +16,8 @@ spec:
   tags:
     # N Version of the image
     - annotations:
-        opendatahub.io/notebook-software: '[{"name":"Python","version":"v3.9"},{"name":"ROCm-PyTorch","version":"2.3"}]'
-        opendatahub.io/notebook-python-dependencies: '[{"name":"ROCm-PyTorch","version":"2.3"},{"name":"Tensorboard","version":"2.16"},{"name":"Kafka-Python","version":"2.0"},{"name":"Matplotlib","version":"3.8"},{"name":"Numpy","version":"1.26"},{"name":"Pandas","version":"2.2"},{"name":"Scikit-learn","version":"1.4"},{"name":"Scipy","version":"1.12"},{"name":"Elyra","version":"3.16"},{"name":"PyMongo","version":"4.6"},{"name":"Pyodbc","version":"5.1"}, {"name":"Codeflare-SDK","version":"0.19"}, {"name":"Sklearn-onnx","version":"1.16"}, {"name":"Psycopg","version":"3.1"}, {"name":"MySQL Connector/Python","version":"8.3"}]'
+        opendatahub.io/notebook-software: '[{"name":"Python","version":"v3.11"},{"name":"ROCM-PyTorch","version":"2.3"}]'
```

Comment:
Is the ROCM-PyTorch verson correct here? In https://quay.io/repository/opendatahub/workbench-images/manifest/sha256:57926cbe6fb0bb8e1a1bd626ea9635ec715cd05f0e4dae91e22cbd05b0f94102?tab=packages, I can see: `torch 2.4.1`.